src/EditorDescriptors/GetaTagsAttribute.cs

```diff
@@ -1,10 +1,12 @@
-﻿using System;
-using System.Linq;
-using System.Web.Mvc;
+﻿using EPiServer.Cms.Shell.Extensions;
 using EPiServer.DataAnnotations;
 using EPiServer.Shell;
 using EPiServer.Shell.ObjectEditing;
 using Geta.Tags.Attributes;
+using Geta.Tags.Helpers;
```

Comment:
Are all usings in use? Please check in other modified files too.

Reply:
All usings are in use, they were inline, so I didn't like that and changed.